src/screens/OrganizationActionItems/ItemDeleteModal.tsx

```
@@ -5,22 +5,22 @@
   isOpen: boolean;
   hide: () => void;
-  actionItem: InterfaceActionItemInfo | null;
+  actionItem: InterfaceActionItem | null;
   actionItemsRefetch: () => void;
 }
 
 /**
  * A modal component for confirming the deletion of an action item.
  *
- * @param props - The properties passed to the component.
+ * @param props: The properties passed to the component.
```

ESLint

[error] tsdoc-param-tag-missing-hyphen: The @param block should be followed by a parameter name and then a hyphen

(tsdoc/syntax)